src/tutorials/Mutable-File-System/09.md

```diff
@@ -0,0 +1,34 @@
+Unlike `files.mv`, which removes items from their source path when moving them to their desination path, the [`files.cp` method](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/FILES.md#filesmv) allows you to a copy a file or directory to a new location while also leaving it intact at its source.
+
+The method looks like this:
+```js
+await ipfs.files.cp(...from, to, [options])
+```
+However, you now have two formatting options for `from`. You can pass in either:
+
+- an existing MFS path to a file or a directory in your own node (e.g. `/my-dir/my-file.txt`)
+- an IPFS path to a file or directory hosted either by you or by a peer (e.g. `/ipfs/QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks`)
+
+Notice that an IPFS path starts with `/ipfs/` and ends with a CID.
+
+As you saw with `files.mv`, `to` is the destination path in MFS, and there's an option `{ parents: true }` that can be used to create parent directories that don't already exist.
+
+You can use `files.cp` to perform a number of different operations:
+
+```js
+// copy a single file into a directory
+await ipfs.files.cp('/source-file.txt', '/destination-directory')
+await ipfs.files.cp('/ipfs/QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks', '/destination-directory')
+
+// copy multiple files into a directory (note the two acceptable formats with or without [ ])
+await ipfs.files.cp('/source-file-1.txt', '/source-file-2.txt', '/destination-directory')
+await ipfs.files.cp(['/source-file-1.txt', '/source-file-2.txt'], '/destination-directory')
+await ipfs.files.cp('/ipfs/QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks',
+ '/ipfs/QmWGeRAEgtsHW3jk7U4qW2CyVy7eA2mFRVbk1nb24jFyre', '/destination-directory')
+await ipfs.files.cp(['/ipfs/QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks',
+ '/ipfs/QmWGeRAEgtsHW3jk7U4qW2CyVy7eA2mFRVbk1nb24jFyre'], '/destination-directory')
+
+// copy a directory into another directory
+await ipfs.files.cp('/source-directory', '/destination-directory')
+await ipfs.files.cp('/ipfs/QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks', '/destination-directory')
+```
```

Comment:
This code block is quite unwieldy - can it be split into multiple blocks?

Reply:
The lesson itself stays looking shorter if we do this as a code block split by comments than if we turn those comments into text and put separate code blocks under each of them. Let's hold off and see if our other testers comment on this. If we change it here then we should change it in lesson 8 as well for consistency.